types/params.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const (
 	// MaxBlockSizeBytes is the maximum permitted size of the blocks.
-	MaxBlockSizeBytes = 104857600 // 100MB
+	MaxBlockSizeBytes = 128 * 1024 * 1024 // 128 MiB
```

Comment:
iirc, this should cause a few test to fail, but many are simple fixes

I think we could get even more benefit out of increasing this to 512MiB or 1GiB or some ridiculous number, that way, we never have to worry about it ever actually enforcing a block validity rule on accident, in which case we would have to pass the app version here if we needed to increase it further

Reply:
A number of questions:

1. This constant is called `MaxBlockSizeBytes` here so why does it differ from: [`MaxBlockBytes`](https://github.com/celestiaorg/celestia-app/blob/defff08dee71949aec42dc4e75a2adf812396581/specs/src/specs/params.md?plain=1#L14) in the specs?
2. Should we bump the value in the specs to match the new value here? 
3. Should we mark this PR as breaking (i.e. with `!`)?
4. Do we plan on including this in a celestia-app v1.x release or only in v2.x?
5. Does a celestia-app block with a square size of 512 actually have a protobuf encoded size less than 128 MiB? I think so but I also recall some other fields in the block occupying some amount of space in the protobuf encoded block so a follow up to this PR could be to actually verify that a square size of 512 can be produced (assuming the other params are also bumped).

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/3322